### DISCUSSION TITLE
selling digital products

### DISCUSSION THREAD

**Discussion by moalim** [woocommerce/woocommerce#51942] [Checkout Flow]
When selling digital products, we do not need the full address details; we only require the country and city. So why can't we make the remaining fields optional?

**Comment by mikejolley** [comment]
The fields are there for calculating taxes and for the payment gateways that require a billing address. See this discussion which links to tutorials for removing more fields:

https://github.com/woocommerce/woocommerce/discussions/45568#discussioncomment-8828548
